tests/models/longformer/test_tokenization_longformer.py

```diff
@@ -28,7 +28,9 @@
 
 
 @require_tokenizers
+# Copied from tests.models.roberta.test_tokenization_roberta.RobertaTokenizationTest with roberta-base->allenai/longformer-base-4096,Roberta->Longformer,roberta->longformer,
 class LongformerTokenizationTest(TokenizerTesterMixin, unittest.TestCase):
+    # ignore copied
```

Comment:
This could be applied to a *block* that is not an real inner block (i.e. not inner class/func).

Reply:
This could not generalize well however: the ignored part (if it is not a real inner block) should have a corresponding block in the target code (i.e. where the copy statement points to).

The main issue here is they don't have real name (i.e. class/func name), and a dummy name is assigned to it.

Not a big deal if this only appear as the block of  class attribute definitions in a class.